bcel-builder/src/main/java/org/aspectj/apache/bcel/util/ClassLoaderRepository.java

```diff
@@ -95,6 +96,16 @@
 
 	public static boolean useSharedCache = System.getProperty("org.aspectj.apache.bcel.useSharedCache", "true").equalsIgnoreCase("true");
 
+	//Cache not found classes as well to prevent unnecessary file I/O operations
+	public static final boolean useUnavailableClassesCache =
+		System.getProperty("org.aspectj.apache.bcel.useUnavailableClassesCache", "true").equalsIgnoreCase("true");
+	//Ignore cache clear requests to not build up the cache over and over again
+	public static final boolean ignoreCacheClearRequests =
+		System.getProperty("org.aspectj.apache.bcel.ignoreCacheClearRequests", "true").equalsIgnoreCase("true");
```

Comment:
This automatically activates the optimisations. Depending on what @aclement wants, the defaults for both properties would have to be `"false"` (or anything else than `"true"`) for backward compatibility. Only if we decide to auto-activate the optimisations, this can stay like it is now.